.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: bash
+services: docker
+
+before_install:
+  - docker-compose up -d
+
+script:
+  - sleep 30 && bash test.sh
```

Comment:
Sleep `30` seconds to wait for the `MySQL` service is executed correctly before doing the test script.